robot/data/variables.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
+
+##
+#    @file    variables.py
+#    @brief   Contains common IPMI constants
+#
+#    @author  Rahul Maheshwari
+#
+#    @date    June 29, 2016
+##
+
+##
+# @brief This class contains common IPMI constants
+#
+class variables():
+    HELP_CMD = "ipmitool -H "
+    PREFIX_CMD = " -I lanplus -U ADMIN -P admin"
```

Comment:
Current op-test code will also work against FSP systems, and we should include them as they are, indeed, still OPAL systems.